src/main/java/com/minh/project/model/Game.java

```diff
@@ -0,0 +1,84 @@
+package com.minh.project.model;
+
+import com.minh.project.model.Game;
+
+public class Game {
+
+	private long id;
```

Comment:
you should use Long instead of the primitive long. When you connect to a DB in the next part of the new joiner app you want the ability for the value to be null